### PR TITLE
Fix BSC bridges

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-1598

### CODE REVIEW

File: src/__swaps__/screens/Swap/providers/swap-provider.tsx [hunk 1]
```diff
@@ -313,13 +314,34 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
...
       sellAmount: quoteData.sellAmount?.toString(),
       buyAmount: quoteData.buyAmount?.toString(),
       chainId: inputAsset.chainId,
       assetToSell: inputAsset,
       assetToBuy: outputAsset,
-      quote: quoteData,
+      quote: {
+        ...quoteData,
+        buyAmountDisplay: isNativeWrapOrUnwrap ? quoteData.buyAmount : quoteData.buyAmountDisplay,
+        sellAmountDisplay: isNativeWrapOrUnwrap ? quoteData.sellAmount : quoteData.sellAmountDisplay,
```
Comment (brunobar79) [inline src/__swaps__/screens/Swap/providers/swap-provider.tsx:340]: what's the logic for overriding these two? I don't think it's necessary.

File: src/__swaps__/screens/Swap/providers/swap-provider.tsx [hunk 2]
```diff
@@ -313,13 +314,34 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
...
+        feeInEth: isNativeWrapOrUnwrap ? '0' : quoteData.feeInEth,
```
Comment (brunobar79) [inline src/__swaps__/screens/Swap/providers/swap-provider.tsx:341]: Also this. The SDK should handle it internally

File: src/__swaps__/screens/Swap/providers/swap-provider.tsx (hunk 1 above)
Comment (walmat) [reply src/__swaps__/screens/Swap/providers/swap-provider.tsx:340]: This was done on the BX so figured it was necessary. If it's not I can remove it

File: src/__swaps__/screens/Swap/providers/swap-provider.tsx (hunk 2 above)
Comment (walmat) [reply src/__swaps__/screens/Swap/providers/swap-provider.tsx:341]: Also brought this over from the BX

File: src/__swaps__/screens/Swap/providers/swap-provider.tsx (hunk 1 above)
Comment (brunobar79) [reply src/__swaps__/screens/Swap/providers/swap-provider.tsx:340]: I can't find anything like that here https://github.com/rainbow-me/browser-extension/blob/ec53546df9bc6ed3adb902d48dc719053ba0e298/src/entries/popup/pages/swap/SwapReviewSheet/SwapReviewSheet.tsx#L293-L304. Am I looking at the wrong place?

**Review by brunobar79 — APPROVED** [review]
LGTM.  Just tested this, working fine.

**Pull request merged**